src/platforms/native/integrations/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Integrations
```

Comment:
Let's rename this to "Backends" (to not confuse it with the actual integrations mentioned in "Guides").